### PR TITLE
make zulu release from q1-2024 wire-build

### PR DESCRIPTION
**Pull request by amitsagtani97**

### CODE REVIEW

**Review by jschumacher-wire — APPROVED** [review]

File: offline/docs_ubuntu_22.04.md [hunk 1]
```diff
@@ -574,6 +574,28 @@ Now deploy `wire-server`:
 d helm install wire-server ./charts/wire-server --timeout=15m0s --values ./values/wire-server/values.yaml --values ./values/wire-server/secrets.yaml
 ```
 
+### Deploying webapp
+
+Update the values in ./values/webapp/prod-values.example.yaml and run
```
Comment (supersven) [inline offline/docs_ubuntu_22.04.md:579]:
```suggestion
Update the values in `./values/webapp/prod-values.example.yaml` and run
```

File: offline/docs_ubuntu_22.04.md [hunk 2]
```diff
@@ -574,6 +574,28 @@ Now deploy `wire-server`:
...
+```
+d helm install webapp ./charts/webapp --values ./values/webapp/prod-values.example.yaml
+```
+
+### Deploying team-settings
+
+Update the values in ./values/team-settings/prod-values.example.yaml and ./values/team-settings/prod-secrets.example.yaml, then run
```
Comment (supersven) [inline offline/docs_ubuntu_22.04.md:586]:
```suggestion
Update the values in `./values/team-settings/prod-values.example.yaml` and `./values/team-settings/prod-secrets.example.yaml`, then run
```

File: offline/docs_ubuntu_22.04.md [hunk 3]
```diff
@@ -574,6 +574,28 @@ Now deploy `wire-server`:
...
+```
+d helm install team-settings ./charts/team-settings --values ./values/team-settings/prod-values.example.yaml --values ./values/team-settings/prod-secrets.example.yaml
+```
+
+### Deploying account-pages
+
+Update the values in ./values/account-pages/prod-values.example.yaml and run
```
Comment (supersven) [inline offline/docs_ubuntu_22.04.md:593]:
```suggestion
Update the values in `./values/account-pages/prod-values.example.yaml` and run
```

**Review by supersven — APPROVED** [review]
LGTM :+1: 

(But please let's wait until the CI is "green")

File: .github/workflows/offline.yml [hunk 4]
```diff
@@ -29,6 +29,9 @@ jobs:
         env:
           GPG_PRIVATE_KEY: '${{ secrets.GPG_PRIVATE_KEY }}'
           DOCKER_LOGIN: '${{ secrets.DOCKER_LOGIN }}'
+          AWS_ACCESS_KEY_ID: '${{ secrets.AWS_ACCESS_KEY_ID }}'
```
Comment (julialongtin) [inline .github/workflows/offline.yml:32]: this should not be required?

**Review by julialongtin — APPROVED** [review]

**Pull request merged**